iib/workers/config.py

```diff
@@ -138,8 +140,8 @@ class DevelopmentConfig(Config):
         ],
     }
     iib_registry = 'registry:8443'
```

Comment:
I see such strings not having static types. Is there a reason for that?

Reply:
I intended to add static types just to method definitions, however I also added static types to class variables. Is it okay in this way?